codejail_service/apps/api/v0/tests/test_views.py

```diff
@@ -86,11 +86,17 @@ def test_unhealthy(self, _mock_is_safe_exec):
             exp_status=500, exp_body={'error': "Codejail service is not correctly configured"},
         )
 
-    def test_success(self):
+    @patch('codejail_service.apps.api.v0.views.set_custom_attribute')
+    def test_success(self, mock_set_custom_attribute):
         """Regular successful call."""
         self._test_codejail_api(
             exp_status=200, exp_body={'globals_dict': {'retval': 7}},
         )
+        assert mock_set_custom_attribute.call_args_list == [
```

Comment:
Did you find `assert_has_calls` later?

Reply:
In this case I intended to assert that this was the complete list of calls. But it's not important -- I could use `assert_has_calls` instead.

Reply:
You can leave as-is. I forgot there isn't an assertion for exact match. Thanks.